controls/block_volume.sp

```diff
@@ -23,16 +23,15 @@ variable "boot_volume_avg_read_write_ops_low" {
 }
 
 locals {
-  block_volume_common_tags = merge(local.thrifty_common_tags, {
-    service = "block_volume"
+  block_volume_common_tags = merge(local.oci_thrifty_common_tags, {
+    service = "OCI/BlockVolume"
```

Comment:
What service do we use in OCI Insights? We should be consistent with that service, otherwise users with this mod + Insights installed will see two different services